packages/utils/src/io.ts

```diff
@@ -184,9 +184,10 @@ export function downloadAndExtractFile(url: string): Promise<FS> {
       dir.set(baseName, content);
     }
 
-    https.get(url, response => {
+    https.get(url, { timeout: 1_000_000 }, response => {
       const extract = tarStream.extract();
       response.pipe(zlib.createGunzip()).pipe(extract);
+      response.on("error", reject);
```

Comment:
further reading suggests that the right place to put this is on the return value of `https.get`, not on `response`. Anybody care to confirm that?